pkg/pool-manager/virtualmachine_pool.go

```diff
@@ -389,42 +396,69 @@ func (p *PoolManager) AddMacToWaitingConfig(allocations map[string]string) error
 	}
 
 	for _, macAddress := range allocations {
-		log.V(1).Info("add mac address to waiting config", "macAddress", macAddress)
-		macAddress = strings.Replace(macAddress, ":", "-", 5)
-		configMap.Data[macAddress] = time.Now().Format(time.RFC3339)
+		if p.IsMacInRange(macAddress) {
```

Comment:
Is it necessary to check it the address is in range as this point? 
The validation is already done right?

Reply:
in AddMacToWaitingConfig() we add new macs allocated to the vmWaitConfigMap.
Since some of them may be in the in range and some don't (manually chosen) - we need to insert only the ones that are in the kmp-mgr range.